bin/basicswap_prepare.py

```diff
@@ -901,7 +971,7 @@ def prepareDataDir(coin, settings, chain, particl_mnemonic, extra_opts={}):
             fp.write('zmqpubsmsg=tcp://{}:{}\n'.format(COINS_RPCBIND_IP, settings['zmqport']))
             fp.write('spentindex=1\n')
             fp.write('txindex=1\n')
-            fp.write('staking=0\n')
+            fp.write('staking=1\n')
```

Comment:
staking was disabled

Reply:
1. Staking is disabled on purpose.
2. What has this got to do with wownero?

Reply:
Ill revert these